eng/Version.Details.xml

```diff
@@ -35,46 +35,6 @@
     <Dependency Name="Microsoft.WindowsAppSDK" Version="0.0.1">
       <Uri>https://dev.azure.com/microsoft/ProjectReunion/_git/ProjectReunionInternal</Uri>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.App.Runtime.win-x64" Version="6.0.3" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
```

Comment:
From my understanding, if we're manually managing the Microsoft.AspNetCore.* dependencies, then we don't need these entries in the Version.Details.xml

Reply:
Added back here. Will setup a subscription (per https://github.com/dotnet/maui/pull/6014#discussion_r849500075) once this PR is merged.